### PR TITLE
Adding the description annotation

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-4822

### CODE REVIEW

**Review by willkutler — APPROVED** [review]

**Comment by dockerymick** [comment]
@willkutler thanks for the review and approval, pulling in a writer for peer review

File: governance/create_policy.adoc [hunk 1]
```diff
@@ -216,6 +217,7 @@ spec:
        cloud: "IBM"
 ----
 
+. (Optional) Add a description for your policy.
```
Comment (oafischer) [inline governance/create_policy.adoc]: Do we want to use our official special tag for this? -> **Optional:**

File: release_notes/whats_new.adoc [hunk 2]
```diff
@@ -42,6 +42,8 @@ For other Application topics, see link:../applications/app_management_overview.a
 
 * Add mulitple YAML strings for your policy templates. Read link:../governance/config_policy_ctrl.adoc#configuration-policy-yaml-table[Configuration policy YAML table] for more details. 
 
+* You can now create, edit, and display descriptions for your policies using the `policy.open-cluster-management.io/description` annotation. Read link:../governance/create_policy.adoc#viewing-your-security-policy-from-the-console[Viewing your security policy from the console].
```
Comment (oafischer) [inline release_notes/whats_new.adoc]:
```suggestion
* You can now create, edit, and display descriptions for your policies by using the `policy.open-cluster-management.io/description` annotation. Read link:../governance/create_policy.adoc#viewing-your-security-policy-from-the-console[Viewing your security policy from the console].
```

**Review by oafischer — APPROVED** [review]
Looks great, left 2 small suggestions

File: governance/create_policy.adoc (hunk 1 above)
Comment (dockerymick) [reply governance/create_policy.adoc]: Yes, Thanks for the reminder !

**Comment by dockerymick** [comment]
@oafischer @willkutler thanks for the reviews and approvals. I made updates based on Oliver's review. Merging

**Pull request merged**